### ISSUE TITLE
Add tests for Member resource

### ISSUE THREAD

**Issue by TheBeege** [CodeSeoul/event_member_management_backend#2]
I haven't decided how exactly we're going to do testing. I'm not sure if we'll only do unit testing, only integration testing, or both. For now, let's just focus on getting one resource tested and the process to do it, then we can focus on adding tests to other resources.

**Comment by xstrengthofonex** [comment]
I'd like to take this on. I haven't used Spring much, but I'm well aware of the pitfalls, particularly where testing is concerned. I wrote a few preliminary tests yesterday, to get a feel for the workflow and landed on using mvcMock for testing the Controller and BDDMockito for mocking out the repository. The tests run relatively quickly, while still giving a good degree of assurance. I found this resources especially useful [Guide to Unit Testing Spring Boot REST APIs](https://stackabuse.com/guide-to-unit-testing-spring-boot-rest-apis/ ). 

Should I put the tests in the EventMemberManagementApplicationTests class or create a new one for MemberControllerTests?
